From c828effdffa232042bd41957c08b10958289d468 Mon Sep 17 00:00:00 2001
From: Nickey Yang <nickey.yang@rock-chips.com>
Date: Mon, 10 Aug 2020 10:40:08 +0800
Subject: [PATCH] ARM: dts: rockchip: rv1109-38-v10-spi-nand: modify rootfs to
 mtd:2

The UBIFS file system does not need misc partition,
So modify the mtd of rootfs to 2.

Signed-off-by: Nickey Yang <nickey.yang@rock-chips.com>
Change-Id: Ic44411fd5251871f768d7192e792190769a92633
---
 arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts b/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts
index db7f9ad85ddb..7f812f7b2729 100644
--- a/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts
+++ b/arch/arm/boot/dts/rv1109-38-v10-spi-nand.dts
@@ -12,7 +12,7 @@
 	compatible = "rockchip,rv1109-38-v10-spi-nand", "rockchip,rv1109";
 
 	chosen {
-		bootargs = "earlycon=uart8250,mmio32,0xff570000 console=ttyFIQ0 ubi.mtd=3 root=ubi0:rootfs rootfstype=ubifs snd_aloop.index=7";
+		bootargs = "earlycon=uart8250,mmio32,0xff570000 console=ttyFIQ0 ubi.mtd=2 root=ubi0:rootfs rootfstype=ubifs snd_aloop.index=7";
 	};
 
 	/delete-node/ vdd-npu;
-- 
2.35.3

